Loop/View Controllers/CarbAbsorptionViewController.swift

```diff
@@ -164,7 +164,7 @@ final class CarbAbsorptionViewController: LoopChartsTableViewController, Identif
                 let review = try await loopDataManager.fetchCarbAbsorptionReview(start: listStart, end: listEnd)
                 insulinCounteractionEffects = review.effectsVelocities.filterDateRange(chartStartDate, nil)
                 carbStatuses = review.carbStatuses
-                carbsOnBoard = carbStatuses?.getClampedCarbsOnBoard()
+                carbsOnBoard = loopDataManager.activeCarbs
                 carbEffects = review.carbEffects
```

Comment:
@ps2 this is where the COB value comes from for the carbs chart. Is it appropriate to re-use in the absorption view?

Reply:
I think this is more a product question.  The `carbStatuses?.getClampedCarbsOnBoard()` should be more current. The `loopDataManager.activeCarbs` is less frequently updated (during a loop, generally).